frontend-training/html-css-training/week_one/01/README.md

```diff
@@ -0,0 +1,45 @@
+# 01 - Introduction to HTML
+
+## Lesson Objectives
+
+After this training stage the trainee should be able to:
+
++ Correctly utilize the basic elements of the Hypertext Markup Language (HTML).
++ Understand the meaning of "markup" and the difference between writing markup and programming.
++ Write simple web pages by using the basic HTML tags and elements.
++ Understand the purposes and functions of different HTML container tags (i.e. `<div>` v.s. `<main>` v.s. `span`).
++ Recognize and apply the fundamental attributes and their values for each of the main HTML elements.
+  
+## Lesson
+
+This lesson consists of a detailed exploration of the HTML syntax, elements and their uses and attributes.
+
++ Demonstrate the overall structure of a static HTML file (and explain why it is considered static)
++ Explain and demo:
+  + Heading Tags: `<h1>`,`<h2>`,`<h3>`, `<h4>`, `<h5>`, `<h6>`
+  + Paragraph Tags: `<p>`
+  + Image Tags: `<img>`
+  + List Tags: `<ul>`, `<ol>`, `<il>`
+  + Layout and Grouping Tags: `<div>`, `<main>`, `<section>`, `<span>`, `<header>`, `<footer>`, etc.
+  + Table Tags: `<table>`, `<thead>`, `<tbody>`, `<th>`, `<tr>`, `<td>`
+  + Form Tags: `<form>`, `<input>`, `<button>`
+
+The [lesson.html](/lesson.html) file in this directory contains commented code examples for all of these concepts. This file can also be opened in the browser to demonstrate the effects of the HTML markup.
+
+## Practice
+
+In order to complete this practice exercise, checkout the [practice.[EXT]](/practice.[EXT]) file for the exercise template.
+
+Each trainee will practice the previously covered theory by writing a Ruby script according to the following instructions/requirements:
+
+## Self Study
+
+### [FREE COURSE LINKS]
```

Comment:
This seems like is part of a template, maybe is a work in progress?

Reply:
Now it's done!

Reply:
Thanks!